stable/akv2k8s/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.1.0
 description: A Helm chart that deploys akv2k8s Controller and Env-Injector to Kubernetes
 name: akv2k8s
-version: 1.1.28
+version: 1.1.29
```

Comment:
Let's add `-beta.1` to the version - then we can test and validate first without making the new version available to everyone.